runner.py

```diff
@@ -95,6 +96,24 @@ def parse_args(targets: Dict[str, TargetBase], hosts: Dict[str, Host]):
         ),
     )
 
+    parser.add_argument(
+        "-S",
+        "--stream",
+        default=True if is_github_actions() else False,
+        action="store_true",
+        help="Stream the DUT output during the test execution. Defaults to %(default)s",
+    )
+
+    parser.add_argument(
+        "-O",
```

Comment:
Found some issue, when asserting `Hello worldd!` in sample helloworld test:
![Screenshot from 2023-08-28 17-35-54](https://github.com/phoenix-rtos/phoenix-rtos-tests/assets/77304431/01d64f71-b54e-40ca-9010-a8e9a0c5d33a)

Reply:
uh, that is a little bit worrying, as the message is an attribute I suppose I need either additional msg esacaping or switch to keeping more detailed information in other tags.

Reply:
I've added some explicit XML escaping, in my tests they should be enough but some further problems might suffice in the future (we will fix it then).

Reply:
Thanks!